_extras/design.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: page
+title: "Lesson Design"
+permalink: /design/
+---
+## Stage 0 - Assumptions
+
+*   Audience
+    *   Graduate students and practicing scientists 
+        *   Materials not specific to discipline
+    *   Must understand basic data management and manipulation 
+        *   Likely worked with spreadsheets and interactive stats tools (SAS)
+    *   Must understand basic statistics (mean, stdev, lm)
+    *   Might have some limited programming experience
+*   Constraints
+    *   One full day 09:00-17:00
+        *   06:30 teaching time
+        *   1:00 for lunch
+        *   0:30 total for two coffee breaks
+    *   Learners use native installs on their own machines
+        *   May use VMs or cloud resources at instructor's discretion
+        *   Must also setup native local install as an option
+    *   May draw from [r-novice-gapminder lesson][r-novice-gapminder]
+        *   How much time do we spend to (re)introduce tidy data structure, `read.csv()`, `dplyr`, `tidyr`?
+    *   Use RStudio for project management and live coding
+*   Data
+    *   Use the [Gapminder data][gapminder-data] throughout
```

Comment:
I'm going to start with the Gapminder data from the r-novice lesson and see if I can make it work. Otherwise, I think I should be able to make it work from a different subset of the Gapminder database.